python/langsmith/schemas.py

```diff
@@ -744,3 +744,96 @@ def metadata(self) -> dict[str, Any]:
         if self.extra is None or "metadata" not in self.extra:
             return {}
         return self.extra["metadata"]
+
+
+class PromptObject(BaseModel):
```

Comment:
renamed to PromptCommit

fetching the prompt with metadata is quite slow since it involves some computations and sometimes you just want the manifest, not the number of downloads or something

pull_prompt fetches the manifest, we could add an arg like "include_metadata" to get all information